docs/photoniq/prerendering/manage-prefetching/configure-prefetching.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: 30
+title: Configure Prefetching
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You have several options for managing Prerendering prefetching configurations.
+
+- **Prefetching tab** - In the Prefetching tab in the web console, you can view and manage prefetching configurations for an origin.
+- **REST API** - Macrometa provides a API calls that allow you to view and manage prefetching configurations for an origin.
+
+## View Prefetching Configurations
+
+<Tabs groupId="operating-systems">
+<TabItem value="console" label="Web Console">
+
+Follow these instructions to view prefetching jobs in the web console.
+
+1. Log in to your Prerendering dashboard.
+2. Click **Prefetching**.
+3. Click **Configuration**.
+
+![Prerendering Prefetching Configuration Tab](/img/prerendering/prefetching-configuration-tab.png)
+
+</TabItem>
+<TabItem value="api" label="REST API">
+
+Use our interactive API Reference with code generation in 18 programming languages to [get prefetching configuration](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin--prefetch-configs/get).
+
+</TabItem>
+</Tabs>
+
+## Prefetching Configuration Fields
+
+Prerendering provides the following prefetching configuration fields. Field names in the web console are different than those in the API responses. Refer to the API endpoint documentation for a full explanation of the API response.
+
+- **Status** - Whether the configuration is active or inactive.
+- **Origin URL** - The origin URL.
+- **Sitemaps** - The sitemap path, if the origin has one.
+- **Scheduling** - Cron expression entered to trigger the prefetching job.
+- **Devices** - Icon indicating whether the prefetch was rendered for just desktop or desktop and mobile.
+- **Actions** - Icon that allows you to update.
+
+## Create Prefetching Configurations
+
+<Tabs groupId="operating-systems2">
+<TabItem value="console" label="Web Console">
+
+A new prefetch configuration appears in the web console when you add a new origin in the Origin Settings tab. For more information about viewing and adding origins, refer to [Manage Origins](../manage-origins/).
```

Comment:
Do we need to say something about the sitemap? As I recall, the sitemap is needed if you are doing prefetching rather than on-demand.

Reply:
The sitemaps column just reminds the user that the prefetching will happen on all URLs in the sitemap.
Those sitemaps are only used when the configuration for the origin is active.